build_tools/python/e2e_test_framework/definitions/iree_definitions.py

```diff
@@ -144,7 +144,7 @@ class MLIRDialectType(Enum):
   """Imported MLIR dialect type."""
   NONE = "none"
   TOSA = "tosa"
-  MHLO = "mhlo"
+  STABLE_HLO = "stablehlo_experimental"
```

Comment:
nit: could qualify the PR title and/or description with _where_ mhlo is being replaced with stablehlo_experimental.

(and then mention any tradeoffs / feature gaps - maybe @kuhar knows of some that would be worth calling out in this PR?)

Reply:
Right now the biggest missing piece is CHLO. If these models don't use chlo ops, then using `stablehlo_experimental` should be fine, although we haven't tested it extensively.

Reply:
Updated

Reply:
Let's make this `stablehlo` instead of `stablehlo_experimental` before we move all the benchmarks to it :-) I know it's just a name, but the name is a signifier of support level

Reply:
I've just submitted a PR for this: https://github.com/openxla/iree/pull/13534

Reply:
... and it's been renamed

Reply:
Updated to use `stablehlo`